### PR TITLE
Add ENV variables support for few configuration option - Closes #2225

### LINKED ISSUE #2225: Add ENV variables support for few configuration option
### Expected behavior
We should specify many configuration option through `ENV` variables. That is required to have a configureable docker image (template file, confd and invocation (in run.sh) must be removed from the docker image)

### Actual behavior
Currently we support some subset of configuration options to be set through the ENV variables. https://github.com/LiskHQ/lisk/tree/1.1.0#command-line-options, which are not enough to run a docker image

### Which version(s) does this affect? (Environment, OS, etc...)
1.1.x

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?
Currently we support some subset of configuration options to be set through the ENV variables. https://github.com/LiskHQ/lisk/tree/1.1.0#command-line-options, which are not enough to run a docker image

### How did I fix it?
Added further support for `ENV` variables. 

### Review checklist

* The PR solves #2225 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,17 +243,32 @@ node app.js [options]
...
+| --peers<br> -p     | LISK_PEERS             | peers.list               | Comma separated list of peers to connect in the format `192.168.99.100:5000,172.169.99.77:5000`                                                                                   |
+|                    | LISK_API_PUBLIC        | api.access.public        | Enable or disable public access of http API. Must be set to true/false                                                                                                            |
+|                    | LISK_API_WHITELIST     | api.access.whiteList     | Comma separated list of IPs to enable API access. Format `192.168.99.100,172.169.99.77`                                                                                           |
+|                    | LISK_FORGING_DELEGATES | forging.delegates        | Comma separated list of delegates to load in the format `publicKey|encryptedPassphrase,publicKey2|encryptedPassphrase2`                                                           |
```
Comment (fchavant) [inline README.md]: I think you have to escape that, it does show properly in the rendered README file.

**Comment by nazarhussain** [comment]
@fchavant Had updated the README with correct formatting.

**Comment by fchavant** [comment]
@nazarhussain it still does not show up as code but it's not mangled at least

**Comment by nazarhussain** [comment]
There was a problem with `nowrap` content in table as well using `|` sign inside table, that causes a lot of trouble. That's the best presentable content I can generate. @fchavant

File: helpers/config.js [hunk 2]
```diff
@@ -90,22 +91,22 @@ function Config(packageJson, parseCommandLineOptions = true) {
 	};
 
 	let commandLineConfig = {
-		wsPort: +program.port || process.env.LISK_WS_PORT || null,
-		httpPort: +program.httpPort || process.env.LISK_HTTP_PORT || null,
-		address: program.address || process.env.LISK_ADDRESS || null,
-		fileLogLevel: program.log || process.env.LISK_FILE_LOG_LEVEL || null,
-		consoleLogLevel: process.env.LISK_CONSOLE_LOG_LEVEL || null,
-		cacheEnabled: process.env.LISK_CACHE_ENABLED === 'true',
+		wsPort: +program.port || getenv(process.env.LISK_WS_PORT),
```
Comment (diego-G) [inline helpers/config.js]: It should be wsPort instead of port.

Comment (nazarhussain) [reply helpers/config.js]: No this is `port` came from this line https://github.com/LiskHQ/lisk/blob/71ab88cdfaf06b71038e3a88644ebab5d2b1441d/helpers/config.js#L50

Didn't change command parameters for backward compatibility.

File: helpers/config.js [hunk 3]
```diff
@@ -88,31 +91,58 @@ function Config(packageJson, parseCommandLineOptions = true) {
 	};
 
 	let commandLineConfig = {
-		wsPort: +program.port || process.env.LISK_WS_PORT || null,
-		httpPort: +program.httpPort || process.env.LISK_HTTP_PORT || null,
-		address: program.address,
-		consoleLogLevel: program.log || process.env.LISK_CONSOLE_LOG_LEVEL,
-		db: { database: program.database },
+		wsPort: +program.port || getenv(process.env.LISK_WS_PORT),
```
Comment (ManuGowda) [inline helpers/config.js]: `LISK_WS_PORT` and `LISK_HTTP_PORT` should be number here?

**Pull request merged**